Documentation/Grafana-Dashboards/ContainerLogV2.json

```diff
@@ -0,0 +1,2533 @@
+              "resources": [
+                "$LAWorkspace"
+              ]
+            },
+            "queryType": "Azure Log Analytics",
+            "refId": "A",
+            "subscription": "E98D0648-F21A-417D-8470-DB17AAB036A7"
+          },
+          "refresh": 1,
+          "regex": "",
+          "skipUrlSync": false,
+          "sort": 0,
+          "type": "query"
+        },
+        {
+          "current": {
+            "selected": true,
+            "text": [
+              "windows-resource-intensive-app5",
+              "test",
+              "windows-resource-intensive-app2",
+              "linux-resource-intensive-app",
+              "windows-resource-intensive-app3",
+              "windows-resource-intensive-app",
+              "linux-resource-intensive-app2",
+              "linux-resource-intensive-app3"
+            ],
+            "value": [
+              "windows-resource-intensive-app5",
+              "test",
+              "windows-resource-intensive-app2",
+              "linux-resource-intensive-app",
+              "windows-resource-intensive-app3",
+              "windows-resource-intensive-app",
+              "linux-resource-intensive-app2",
+              "linux-resource-intensive-app3"
+            ]
+          },
+          "datasource": {
+            "type": "grafana-azure-monitor-datasource",
+            "uid": "azure-monitor-oob"
+          },
+          "definition": "",
+          "hide": 0,
+          "includeAll": false,
+          "multi": true,
+          "name": "LabelValue",
+          "options": [],
+          "query": {
+            "azureLogAnalytics": {
+              "query": "ContainerLogV2\r\n| where $__timeFilter(TimeGenerated)\r\n| where _ResourceId =~ \"/subscriptions/$Subscription/resourcegroups/$rg/providers/Microsoft.ContainerService/managedClusters/$Cluster\"\r\n| where  PodNamespace == \"$Namespace\"\r\n| where isnotempty(KubernetesMetadata)\r\n| extend LabelsJson = KubernetesMetadata.podLabels\r\n| mv-expand LabelsJson\r\n| extend LabelName = tostring(bag_keys(LabelsJson)[0])\r\n| where LabelName in ($LabelKey)\r\n| extend LabelValue = tostring(LabelsJson[LabelName])\r\n| where isnotempty(LabelValue)\r\n| distinct LabelValue",
+              "resources": [
+                "$LAWorkspace"
+              ]
+            },
+            "queryType": "Azure Log Analytics",
+            "refId": "A",
+            "subscription": "E98D0648-F21A-417D-8470-DB17AAB036A7"
+          },
+          "refresh": 1,
+          "regex": "",
+          "skipUrlSync": false,
+          "sort": 0,
+          "type": "query"
+        }
+      ]
+    },
+    "time": {
+      "from": "now-1h",
+      "to": "now"
+    },
+    "timepicker": {},
+    "timezone": "",
+    "title": "Azure Monitor / Container Insights / ContainerLogV2 LW multiselect",
+    "uid": "c1ea0a58-c330-4abf-94a7-821bb0d5c20d",
```

Comment:
Update from "ContainerLogV2 LW multiselect" to "ContainerLogV2"

Reply:
updated